### PR TITLE
API works from appointments controller & model; fixed time to accept …

### PR DESCRIPTION
**Pull request by l1ngoman**
…other days than today

### CODE REVIEW

File: app/controllers/appointments_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class AppointmentsController < ApplicationController
   before_action :set_appointment, only: [:show, :update, :destroy]
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
```
Comment (bwanicur) [inline app/controllers/appointments_controller.rb]: don't forget to turn this back on.   you can add to this PR by creating a new commit in this branch and pushing to github.  Then ping me by using the "@" symbol, so I get an email notification.

File: app/models/appointment.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
 class Appointment < ActiveRecord::Base
   validates :name, presence: true
   validates :phone_number, presence: true
-  validates :time, presence: true
+  validates :exactTime, presence: true
```
Comment (bwanicur) [inline app/models/appointment.rb]: does this work ?   typically we want the column names to be `exact_time` unless there is a REALLY good reason to break this convention.

File: db/migrate/20181213212302_fix_column_name.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class FixColumnName < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :appointments, :time, :exactTime
```
Comment (bwanicur) [inline db/migrate/20181213212302_fix_column_name.rb]: same comment as above --> `exact_time`

**Comment by l1ngoman** [comment]
@bwanicur all fixed! Please review again.

**Pull request merged**